conda-forge.yml

```diff
@@ -3,5 +3,6 @@ appveyor:
 compiler_stack: comp7
 max_py_ver: '37'
 max_r_ver: '35'
+provider: {linux_aarch64: azure, linux_ppc64le: azure}
```

Comment:
```suggestion
provider: {linux_aarch64: default, linux_ppc64le: default}
```

Reply:
This way we use native builds

Reply:
But native  vs qemu doesn't seem to be the problem. This seems to be a cos6/cos7 graphics stack issue. Try adding libegl-devel cdt. That one exists for ppc64le but not yet for aarch64. PR for that one needs reviewing.